cmd/keytransparency-server/frontend.go

```diff
@@ -202,7 +202,7 @@ func main() {
 	mutator := entry.New()
 
 	// Connect to log server.
-	tconn, err := grpc.Dial(*logURL)
+	tconn, err := grpc.Dial(*logURL, grpc.WithInsecure())
```

Comment:
Should we add a flag specifying if security is needed? I'm thinking of the scenario where Trillian and Key Transparency servers are not on the same machine.

There's no need to address this in this PR. If you think it's a good idea to have such a flag, we should open an issue.